src/project/manifest/mod.rs

```diff
@@ -506,6 +507,16 @@ impl Manifest {
             .any(|f| f.pypi_dependencies.is_some())
     }
 
+    /// Returns what pypi mapping configuration we should use.
+    /// It can be a custom one  in following format : conda_name: pypi_name
+    /// Or we can use our self-hosted
+    pub fn custom_pypi_mapping(&self) -> MappingSource {
```

Comment:
Maybe we can just call this `pypi_name_mapping_source`?

Reply:
renamed!